src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -197,7 +197,7 @@ const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
         aboutTags={aboutTags}
         imageLocator={promoImage}
       />
-      <Main role="main">
+      <Main role="main" id="content" tabIndex={-1}>
```

Comment:
can we use a string value like you would in a normal html attribute?

Reply:
Yes, we can use a string but according to the react docs it should be an integer.